lib/jekyll/filters.rb

```diff
@@ -260,6 +261,23 @@ def where_exp(input, variable, expression)
       end
     end
 
+    # Convert the input into integer
+    #
+    # input - the object string
+    #
+    # Returns the integer value
+    def to_integer(input)
+      return input if input.is_a?(Integer)
```

Comment:
This is uncessary, Ruby is already optimized for this and just slows down to_i already.